e2e/_suites/fleet/fleet.go

```diff
@@ -419,12 +419,7 @@ func (fts *FleetTestSuite) processStateChangedOnTheHost(process string, state st
 func (fts *FleetTestSuite) setup() error {
 	log.Trace("Creating Fleet setup")
 
-	err := createFleetConfiguration()
```

Comment:
I think we can remove the entire function, as it's not used elsewhere

Reply:
No, it's not possible to remove this line, as the initialisation of the fleet admin users is performed by that API call.